cargo-paradedb/src/fixtures/postgres_test_client.rs

```diff
@@ -0,0 +1,208 @@
+#![allow(dead_code)]
+
+// Copyright (c) 2023-2024 Retake, Inc.
+//
+// This file is part of ParadeDB - Postgres for Search and Analytics
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+// TECH DEBT: This file is a copy of the `db.rs` file from https://github.com/paradedb/paradedb/blob/dev/shared/src/fixtures/db.rs
+// We duplicated because the paradedb repo may use a different version of pgrx than pg_analytics, but eventually we should
+// move this into a separate crate without any dependencies on pgrx.
```

Comment:
You can remove this file, then. We already enforce that we always use the same `pgrx` version. In the near future, we'll move `cargo-paradedb` to its own repository as well, so this won't be necessary.